config/sidekiq.yml

```diff
@@ -23,6 +23,9 @@
   TransferChannelsJob:
     cron: "0 8 * * *"
     description: "Complete transfer of channels that have exceeded their timeout time without being rejected"
+  Sync::ChannelPromoRegistrationsStatsJob:
+    cron: "0 1 * * *"
+    description: "Sync stats for channel owned referral codes with the promo server every hour."
```

Comment:
This cron config does every day at 1:00, I think the previous interval was each hour, consider using `0 * * * *` which covers each hour